Assets/Scripts/ssjAug2018/World/Mailbox/MailboxManager.cs

```diff
@@ -73,8 +71,8 @@ public void ActivateMailboxGroup()
 
             // Get valid boxes for the set
             HashSet<Mailbox> validBoxes = new HashSet<Mailbox>();
-            Collider[] allBoxes = Physics.OverlapSphere(_player.transform.position, _mailboxData.PlayerMaxRange, LayerMask.GetMask("Mailboxes"));
-            Collider[] ignoreBoxes = Physics.OverlapSphere(_player.transform.position, _mailboxData.PlayerMinRange, LayerMask.GetMask("Mailboxes"));
+            Collider[] allBoxes = Physics.OverlapSphere(player.transform.position, _mailboxData.PlayerMaxRange, LayerMask.GetMask("Mailboxes"));
```

Comment:
Not a requirement, but look into OverlapSphereNonAlloc. it'll save some memory and is something I should be using for my stuff as well.